### PR TITLE
修改

### PR DESCRIPTION
**Pull request by chenwangnatsukashii**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -784,21 +784,21 @@ Contributions are always welcome! Make an individual pull request for each sugge
 
 
 
-*Junbo Zhang, Yu Zheng, Junkai Sun, Dekang Qi*
+*Jinlei Zhang, Feng Chen, Yinan Guo, Xiaohong Li*
```
Comment (SuperSupeng) [inline README.md:787]: 这里修改有误

Comment (SuperSupeng) [inline README.md:787]: 文章链接 https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=8606218

**Review by SuperSupeng — COMMENTED** [review]
修改一下这部分内容

File: README.md (hunk 1 above)
Comment (chenwangnatsukashii) [reply README.md:787]: Multi-Graph Convolutional Network for Short-Term Passenger Flow Forecasting in Urban Rail Transit.
https://arxiv.org/pdf/2001.07512.pdf

**Pull request merged**